assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -256,8 +271,7 @@ def run(args):
             return
 
         baseline_df = baseline_df.toPandas()
-        task_type = args.task_type if args.task_type else determine_task_type(args.target_column, baseline_df)
```

Comment:
You already have the None Check for the passed-in args parameter. but it still crashes, i believe the None is returned from determine_task_type(args.target_column, baseline_df)

Reply:
right I added more datatypes / better checking for each kind of datatype, plus default to classification in the catch all scenario